clang/test/SemaSYCL/intel-reqd-work-group-size-ast-device.cpp

```diff
@@ -142,48 +118,6 @@ int main() {
     FunctorAttr fattr;
     h.single_task<class kernel_name4>(fattr);
 
-    // CHECK: FunctionDecl {{.*}} {{.*}}kernel_name5
-    // CHECK: SYCLReqdWorkGroupSizeAttr
-    // CHECK-NEXT:  ConstantExpr{{.*}}'int'
-    // CHECK-NEXT:  value: Int 32
-    // CHECK-NEXT:  IntegerLiteral{{.*}}32{{$}}
-    // CHECK-NEXT:  ConstantExpr{{.*}}'int'
-    // CHECK-NEXT:  value: Int 32
-    // CHECK-NEXT:  IntegerLiteral{{.*}}32{{$}}
-    // CHECK-NEXT:  ConstantExpr{{.*}}'int'
-    // CHECK-NEXT:  value: Int 32
-    // CHECK-NEXT:  IntegerLiteral{{.*}}32{{$}}
-    h.single_task<class kernel_name5>([]() [[sycl::reqd_work_group_size(32, 32, 32)]] {
-      f32x32x32();
-    });
```

Comment:
It seems like without this piece of code we won't have a test where we check for AST based on the attribute that directly applies to a lambda. I would suggest dropping `f32x32x32` call from the lambda, but keeping the "test case" itself.